pyanaconda/install.py

```diff
@@ -130,8 +130,17 @@ def doConfiguration(storage, payload, ksdata, instClass):
     else:
         _writeKS(ksdata)
 
-    # write out the user interaction config file
-    screen_access.sam.write_out_config_file()
+    # Write out the user interaction config file.
+    #
+    # But make sure it's not written out in the image and directory installation mode,
+    # as that might result in spokes being inadvertedly hidden when the actual installation
+    # startes from the generate image or directory contents.
```

Comment:
"*starts*". But I must say I don't really understand this. Shouldn't the configuration file go to the installed image or directory? I don't think we can rely on some assumption that every image or directory installation is then used to run installation again.